README.md

```diff
@@ -4,28 +4,37 @@
 
 This project provides a Java agent JAR that can be attached to any Java 7+
 application and dynamically injects bytecode to capture telemetry from a
-number of popular libraries and frameworks. The telemetry data can exported
-in a variety of formats each provided as their own independent JAR. In
-addition, the agent and exporter can be configured via command line arguments
+number of popular libraries and frameworks. 
+The telemetry data can be exported in a variety of formats. 
+In addition, the agent and exporter can be configured via command line arguments
 or environment variables. The net result is the ability to gather telemetry
 data from a Java application without code changes.
 
 ## Getting Started
 
-Download the [latest
-release](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases)
-of the Java agent and available exporters.
+[ ![Download](https://api.bintray.com/packages/open-telemetry/maven/opentelemetry-java-instrumentation/images/download.svg) ](https://bintray.com/open-telemetry/maven/opentelemetry-java-instrumentation/_latestVersion)
+
+The easiest way to start is to download "all-in" package, which includes instrumentation agent,
+instrumentations for all supported libraries and all available data exporters.
+This provides completely automatic out of the box experience.
+TODO add here screenshot from Bintray download list.
```

Comment:
This seems a bit complicated. Can we provide a direct link to download the `-all` package?

Reply:
The best I can do now is https://www.jfrog.com/confluence/display/BT/Managing+Uploaded+Content#ManagingUploadedContent-CreatingandEditingaDownloadList. Have a link in documentation to the latest version's screen and there is "Download list" which has our `-all` artefact. I asked JFrog if we can have direct link to the latest version of a specific file.